spec/granite/query/assemblers/mysql_spec.cr

```diff
@@ -23,6 +23,16 @@ require "../spec_helper"
         assembler.where
         assembler.numbered_parameters.should eq ["bob", "23"]
       end
+
+      it "property defines IN query" do
```

Comment:
In the case of Ints, Floats and bools would it have single quotes? I ask because I don't see tests for those cases.

Reply:
Ah crap.  No they wouldn't.  I'll get that fixed up.

Reply:
@eliasjpr Fixed.  This did expose an issue with `Float32` types.  I'll look into it later and make a separate PR for that.  Not sure if its a Granite issue or DB/adapter shard issue yet.